scripts/config/index-rules.conf

```diff
@@ -0,0 +1,12 @@
+# This config file controls when to prune metrics from the index
+# Note:
+# * This file is optional. If it is not present, we won't prune data
+# * Patterns are regexes matched on the metric name (including tags) and tried from top to bottom. First match wins.
+# * Anything not matched or resolving to max-stale=0 will not be pruned
+# * Patterns are unanchored regular expressions; add '^' or '$' to match the beginning or end of a pattern
+# * max-stale is a duration like 7d. if no new data has been seen for a metric during this time, it will be pruned
+# * Valid units are s/sec/secs/second/seconds, m/min/mins/minute/minutes, h/hour/hours, d/day/days, w/week/weeks, mon/month/months, y/year/years
+
+[default]
+pattern = 
```

Comment:
should this have a match-all pattern?

Reply:
empty string is a valid regex that matches everything. (the first test in conf/indexrules_test.go checks this)